apps/core-ie/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "core-ie",
+  "version": "0.0.0",
+  "description": "",
+  "license": "MIT",
+  "scripts": {
+    "start": "lite-server"
+  },
+  "dependencies": {
+    "@clr/city": "^1.1.0",
+    "@clr/core": "./packages/core/dist/core",
+    "normalize.css": "^8.0.1"
```

Comment:
nit: seems like we are using cdn for including `normalize.css` in this app so this dependency is not needed?

Reply:
Good catch! I'll get this in my follow up bugfix pr